### ISSUE TITLE
No prompt for username when attempting remote server

### ISSUE THREAD

**Issue by mcburneym** [microsoft/vscode-jupyter#11794]
### Applies To

- [X] Notebooks (.ipynb files)
- [ ] Interactive Window and\/or Cell Scripts (.py files with \#%% markers)

### What happened?

In VS Code when attempting to connect to a remote server I am promted for the URI but no prompt for username. It goes immediately to asking for a password. 

### VS Code Version

1.72.2

### Jupyter Extension Version

2022.9.120

**Comment by mcburneym** [comment]
For clarity, I am trying to connect to this remote server - https://aafc.maps.arcgis.com/
This is a subscription based ESRI product for which we have an account

**Comment by DonJayamanne** [comment]
@mcburneym I'm sorry for not getting back to you, are you still running into this iussue

**Comment by mcburneym** [comment]
> @mcburneym I'm sorry for not getting back to you, are you still running into this iussue

Yes I am. If i switch attempt to switch from Jupyter Server: Local to Remote and put in the server referenced above I am prompted for a Password but no request for a username.
